src/main/java/com/sesac/carematching/user/User.java

```diff
@@ -0,0 +1,59 @@
+package com.sesac.carematching.user;
+
+import com.sesac.carematching.user.role.Role;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.ColumnDefault;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import org.springframework.data.annotation.CreatedDate;
+
+import java.time.Instant;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "USER")
+public class User {
+    @Id
+    @Column(name = "UNO", nullable = false)
+    private Integer id;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @OnDelete(action = OnDeleteAction.CASCADE)
```

Comment:
다만, Role의 레코드가 삭제할 때, 연관된 User가 전부 사라지면 곤란하니, 현재 지우려는 Role과 연관된 User가 존재한다면 지워지지 않도록 Role에는 OnDelete에 CASCADE 옵션을 지우는게 좋을 것 같음. (Role을 지우기 전에 연관된 User의 Role을 다른 Role로 UPDATE 해주자)

```suggestion
    @ManyToOne(fetch = FetchType.LAZY, optional = false, cascade = CascadeType.RESTRICT)
```

이렇게?